### PR TITLE
Deployment page

### PR DESCRIPTION
**Pull request by nik-dange**
Completed the deployment page. Lmk if anything else needs to be clarified or added

### CODE REVIEW

File: pages/hack-school/deployment.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Deployment is the process of publishing your code on a server so that it becomes accessible over the internet. If you develop a project, your computer can act as a server, but all user interaction might be locally limited. Production-level applications are all deployed on servers, enabling multiple
+users to access and interact with them.
+
+## How do servers (briefly) work?
+
+Recalling back to our discussion on clients and servers in week 4, clients send requests to servers for data, and servers process these requests and send back data in the form of a response.
+Formally, a server is a specialized computer that can accept requests made by clients across a network.
+
+Servers are linked to a network via **IP addresses**. IP addresses are unique identifiers (similar to a home address), allowing distinction between servers. This IP address is where data packets are sent (from the client's requests). Learn more about
+IP addresses here.
```
Comment (AVERGNET) [inline pages/hack-school/deployment.mdx:14]: There should be a link here?

File: pages/hack-school/deployment.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## What is deployment?
+
+Deployment is the process of publishing your code on a server so that it becomes accessible over the internet. If you develop a project, your computer can act as a server, but all user interaction might be locally limited. Production-level applications are all deployed on servers, enabling multiple
+users to access and interact with them.
+
+## How do servers (briefly) work?
+
+Recalling back to our discussion on clients and servers in week 4, clients send requests to servers for data, and servers process these requests and send back data in the form of a response.
+Formally, a server is a specialized computer that can accept requests made by clients across a network.
+
```
Comment (AVERGNET) [inline pages/hack-school/deployment.mdx:12]: Maybe a picture in this sectino could help for understanding

**Review by AVERGNET — COMMENTED** [review]
Looks solid just maybe adding a picture and missing link

File: pages/hack-school/deployment.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Next, you can configure certain settings. For example, if you have both a `client` and `server` directory in your repository, you'll want to change the
+Root Directory to point to the one with front-end code. You can also specify any build commands or environment variables at this stage.
+
+![Vercel configurations](pages/hack-school/images/vercel_config.png)
+
+
+If you want to change these settings later on, they can be found under the specific project settings page.
+
+Finally, deploy your project! You should see this interface, where you can view active branches, the status, logs, and more.
```
Comment (khushijpatel) [inline pages/hack-school/deployment.mdx:37]: Maybe add an image showing the interface

**Review by khushijpatel — APPROVED** [review]
lgtm

**Pull request merged**